modules/site-navigation/data/endpoints/homepage.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Elementor\Modules\SiteNavigation\Data\Endpoints;
+
+use Elementor\Data\V2\Base\Endpoint;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Homepage extends Endpoint {
+
+	public function get_permission_callback( $request ) {
+		return current_user_can( 'edit_posts' );
+	}
+
+	public function get_name() {
+		return 'homepage';
+	}
+
+	public function get_format() {
+		return 'site-navigation/homepage';
+	}
+
+	public function get_items( $request ) {
+		$homepage_id = get_option( 'page_on_front' );
+		$show_on_front = get_option( 'show_on_front' );
+
+		if ( 'page' !== $show_on_front ) {
+			return 0;
+		}
+
+		return intval( $homepage_id );
```

Comment:
return 'page' !== $show_on_front ? 0 : intval( $homepage_id )

Reply:
Be positive instead 😉

```PHP
return 'page' === $show_on_front ? intval( $homepage_id ) : 0;
```